### PR TITLE
Adding systemd scripts

### PR DESCRIPTION
**Pull request by DoriftoShoes**

### CODE REVIEW

File: manifests/helper/systemd.pp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  $process_type = 'single'
+  ) {
+
+  if $process_type == 'multi' {
+    $type = "${process_type}@"
+  } else {
+    $type = ''
+  }
+
+  file{"/etc/systemd/system/${st2_process}${type}.service":
```
Comment (jfryman) [inline manifests/helper/systemd.pp]: You'll want to be careful here, and make sure this works with `st2ctl`.

Comment (DoriftoShoes) [inline manifests/helper/systemd.pp]: I tweaked it to only enable the single process ones.  This should avoid any issues with the @ ones vs the singles.

File: files/systemd/system/st2web.service [hunk 2]
```diff
@@ -0,0 +1,16 @@
+description "Start & stop st2web"
```
Comment (jfryman) [inline files/systemd/system/st2web.service]: This looks more like an upstart script

File: manifests/helper/service_manager.pp [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  $init_provider = $::st2::params::init_type
+
+  if $osfamily == 'Debian' {
+
+    file { "/etc/init/${st2_process}.conf":
+      ensure => present,
+      owner  => 'root',
+      group  => 'root',
+      mode   => '0444',
+      source => "puppet:///modules/st2/etc/init/${st2_process}.conf",
```
Comment (jfryman) [inline manifests/helper/service_manager.pp]: `content => template('st2/...')`

File: manifests/helper/service_manager.pp [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+          process_type => $process_type
+        }
+      } else {
+        $process_type = 'single'
+        st2::helper::systemd{ "${st2_process}_systemd":
+          st2_process  => $st2_process,
+          process_type => $process_type
+        }
+      }
+    } elsif $operatingsystemmajrelease == '6' {
```
Comment (jfryman) [inline manifests/helper/service_manager.pp]: Cool. Thanks for the stub.

File: files/systemd/system/st2web.service (hunk 2 above)
Comment (DoriftoShoes) [inline files/systemd/system/st2web.service]: oops.  Fixing.

File: manifests/helper/service_manager.pp (hunk 3 above)
Comment (DoriftoShoes) [inline manifests/helper/service_manager.pp]: https://github.com/StackStorm/puppet-st2/tree/master/files/etc/init

Those aren't templatized.

Comment (jfryman) [inline manifests/helper/service_manager.pp]: ah! my bad.

**Pull request merged**